pandas/tests/frame/test_replace.py

```diff
@@ -1070,17 +1070,27 @@ def test_replace_truthy(self):
         assert_frame_equal(r, e)
 
     def test_replace_int_to_int_chain(self):
```

Comment:
I guess could update the test titles to be a little clearer. Something like `test_nested_dict_overlapping_replace_int` works (and _str) for below.

Reply:
ok! i changed to `test_nested_dict_overlapping_keys_replace_int` @WillAyd